README_template.md

```diff
@@ -0,0 +1,51 @@
+<!-- markdownlint-disable MD041 -->
```

Comment:
You need to move all content from `README_template.md` to `README.md` and vice versa